### PR TITLE
Add AppVeyor CI badge and configuration

### PR DESCRIPTION
**Pull request by jan-cerny**
This will add:
* an AppVeyor icon which will show the status of last build on AppVeyor, where we build WIndows builds.
* AppVeyor project configuration in YAML format

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+version: 1.0.{build}
```
Comment (mpreisler) [inline appveyor.yml]: why not just {build}?

Comment (mpreisler) [reply appveyor.yml]: or master-{build} or something like that. The 1.0 is confusing.

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by mpreisler** [comment]
@jan-cerny could you set up appveyor not to post comments but instead set commit status? similar to what jenkins does? posting comments will blow up my notifications.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**